DXRModules/DeusEx/Classes/DXRBingoCampaign.uc

```diff
@@ -129,6 +129,22 @@ function FirstEntry()
     }
 }
 
+function PostFirstEntry()
+{
+    switch (dxr.localURL) {
+        case "04_NYC_STREET":
+        case "04_NYC_HOTEL":
+        case "04_NYC_BatteryPark":
+            HandleSpecialPerson("AnnaNavarre", "AnnaNavarre_DeadM5");
```

Comment:
`if (IsBoardGoal(goal)) return;` is looking for `AnnaNavarre_DeadM5` but it should be looking for `AnnaNavarre_DeadM4` on the current board, there will need to be 2 separate arguments for current_goal and future_goal